bigtable_automation/gcf/controller_trigger.go

```diff
@@ -102,3 +103,30 @@ func (s CustomDCPubSubMsg) Publish(ctx context.Context, p PublishConfig) error {
 	log.Printf("PubSub import notification published with server generated msg id: %v\n", id)
 	return nil
 }
+
+// Find the root directory for data imports on gcs.
+// For data import directory structures, please refer to
+// https://docs.datacommons.org/custom_dc/upload_data.html
+//
+// Root directory in the above example is "gcs_folder"
+// Example:
+// findRootImportDirectory("/somepath/gcs_folder/internal/control/trigger.txt")
+// returns "/somepath/gcs_folder/"
+//
+// Note manifest generator requires the "/" at the end.
```

Comment:
Maybe start to add test, can use mock GCS as suggested in https://stackoverflow.com/questions/39814108/how-to-unit-test-google-cloud-storage-in-golang

Reply:
Will leave this as a todo